src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+  public static void main(String[] args) {
+    Machine[] machines = new Machine[]{
```

Comment:
I consider it's better create separate objects for each machine and set by index, to make the code more clearly.